service/view_building_mutation.cc

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (C) 2025-present ScyllaDB
+ */
+
+/*
+ * SPDX-License-Identifier: LicenseRef-ScyllaDB-Source-Available-1.0
+ */
+
+#include "service/view_building_mutation.hh"
```

Comment:
Again, I'd ask to please a comment in the top of the file explaining its scope. I had a vague guess as to what a "view building mutation" might be, the code I see below isn't it. So what is a "view building mutation", or a "view building task mutation builder"?

There's a discussion in the comments whether this uses or doesn't use or can't use or maybe will use in the future other "mutation building" mechanisms in the code, the comment can explain this as well, why view building doesn't use or can't use other code and needs this.

Reply:
Reading more below, I see my vague guess was completely wrong - it seems this code is not about building the view mutations in the view builder - it's apparently about building the view builder's internal table of marking its progress. Which seems to be duplicate of similar code I saw in previous patches (that code used CQL commands). I don't know if this is good or bad or will need to be cleaned up in a future patch, but in the very least we need a comment explaining what this file is about.

Reply:
Yes, somewhere in the PR I wrote that I firstly created methods in `system_keyspace` to manage view building tasks (which return `future<>` and produce multiple mutations when multiple tasks are created/changed/dropped).

But later I've introduced this `view_building_task_mutation_builder` to produce one mutation per batch (`system_keyspace.view_building_tasks` is single partition table) and to do it without async (useful when attaching to tablet operations which use similar builder).

And I'm going to remove the system keyspace methods in favor of this builder.

Added a comment and changed the file name to `view_building_task_mutation_builder`